Plugin fails to report errors which are reported by running mypy on command line

<!--
Any HTML comment will be stripped when the markdown is rendered, so you don't need to delete them.

Put an x inside the [] like this: [x] to mark the checkbox.
-->
### Step 1: Are you in the right place?
- [X] I have verified there are no duplicate active or recent bugs, questions, or requests
- [X] I have verified that I am using the latest version of the plugin.

### Step 2: Describe your environment
 - Plugin version: `0.10.2`
 - PyCharm/IDEA version: `Community Edition 2018.2`
 - Mypy version: `0.630` (installed in venv folder automatically by plugin)

### Step 3: Describe the problem:

I'm encountering cases where the plugin is failing to list mypy errors in the error list window, but running mypy on the command line shows the errors.

I'm getting this in a large and complex proprietary project that I cannot paste the source of, but I have reproduced similar behaviour in a small test project

#### Steps to reproduce:

  1.  Make a new project   File->New Project
  2.  Make two new python files in the project, `File1.py` and `File2.py`
  3.  Add the following contents to the files

in File1.py:

```
class MyClass:

    x:int
    y:str
```

in File2.py:

```
import File1

from typing import List

def SomeFunction( stuffs : List[File1.MyClass] ) -> None:

    a = File1.MyClass()
    a.foo = "bar"

    b = 1
    b = "2"

    for stuff in stuffs:
        stuff.foo = "bar"
        stuff["hello"] = "goodbye"
```

* With `File2.py` open, press the "Check Current File" button in in the mypy plugin window
* On the command line, 

<!-- 
What happened?  This could be a description, log output, etc.
-->
#### Observed Results:

On the command line, I get the following errors from the same mypy executable that the plugin installed for me automatically:

```
PS C:\Users\admin\PycharmProjects\Test> C:\Users\admin\PycharmProjects\Test\venv\Scripts\mypy.exe .\File2.py
File2.py:9: error: "MyClass" has no attribute "foo"
File2.py:12: error: Incompatible types in assignment (expression has type "str", variable has type "int")
File2.py:15: error: "MyClass" has no attribute "foo"
File2.py:16: error: Unsupported target for indexed assignment
```

but in the mypy plugin window I only get one of the errors, only the one about the variable `b`
  
![image](https://user-images.githubusercontent.com/2461007/46169583-6c99ef80-c250-11e8-907e-0166fdb9862d.png)

None of the errors related to the `MyClass` defined in `File1.py` appear.

When I run using the "Check Module" button in the plugin I do successfully get all four errors in this case

![image](https://user-images.githubusercontent.com/2461007/46169650-98b57080-c250-11e8-91ec-ea90346d6922.png)

However, I'm unsatisfied with this for two reasons:

* The command line above is only checking the single file, and is still able to detect errors related to the `MyClass` type even though it is defined externally, so why should the plugin not be able to?  
* It's hard to know what the plugin is doing when I press the "Check Current File" button that's different from what I'm doing on the command line (see Issue #20 )
* In my larger, more complicated project, I'm getting errors that command-line mypy detects but neither the "Check Current File" operation nor the "Check Module" operation in the plugin list, and it's not clear why.

--- a/src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java
+++ b/src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java
@@ -280,7 +280,7 @@ public class MypyRunner {
             cmd.addParameter("--show-column-numbers");
         }
         cmd.addParameter("--follow-imports");
-        cmd.addParameter("skip");
+        cmd.addParameter("silent");
 
         injectEnvironmentVariables(project, cmd);
 

